Dockerfile.rocm

```diff
@@ -129,6 +154,11 @@ if ls /install/*.deb; then \
     && sed -i 's/, hipblaslt-dev \(.*\), hipcub-dev/, hipcub-dev/g' /var/lib/dpkg/status \
     && sed -i 's/, hipblaslt \(.*\), hipfft/, hipfft/g' /var/lib/dpkg/status; \
 fi
+# Install pytorch
```

Comment:
I think we can remove the `hipBLASLt` install before this because our FP8 no longer needs it.

Reply:
gradlib and tuned gemm need it

Reply:
I hate to say it, but they work for me without installing `hipBLASLt` here. And it has always worked.

The old FP8 implementation seems to have used some hipBLASLt API that is unstable, whereas gradlib does not.